app/indexers/facet_indexer.rb

```diff
@@ -48,6 +49,14 @@ def structure?
     false
   end
 
+  def pub_date_start
+    date = resource.primary_imported_metadata.created
+    return unless date
```

Comment:
Should this be `date.present?`? I assume created is a Set and could be an empty array.

Reply:
I added a test where the source doesn't have the created date and it goes okay. but `present?` works too and is more flexible so 👍